spec/features/private_metrics_spec.rb

```diff
@@ -1,11 +1,19 @@
-# frozen_string_literal: true
-
 require 'rails_helper'
 
-RSpec.describe 'Private metrics', type: :feature do
-  it 'works' do
-    expect {
-      visit api_private_metrics_path
-    }.not_to raise_error
+RSpec.describe 'private metrics', type: :feature do
+  let!(:user_with_token)              { create(:user, :with_jwt) }
+  let!(:user_with_inactive_token)     { create(:user, :with_jwt) }
+  let!(:user_without_token)           { create(:user) }
+
+  it 'works (dont want to test images generated by chartkick)' do
+    allow_any_instance_of(UsedJwtIdsElasticQuery).to receive(:perform).and_return(
+      user_with_inactive_token.jwt_api_entreprise.pluck(:id)
+    )
+
+    allow_any_instance_of(NotInProductionJwtIdsElasticQuery).to receive(:perform).and_return(
+      user_with_inactive_token.jwt_api_entreprise.pluck(:id)
+    )
+
+    expect{ visit('/api/admin/private_metrics') }.not_to raise_error
```

Comment:
j'utiliserai le helper de route ici, vu que je ne suis pas sûr que visit une URL invalid lève une erreur (ça serait plus un 404)

```suggestion
    expect{ visit(api_private_metrics_path) }.not_to raise_error
```